src/redux/Checkout/reducers.js

```diff
@@ -297,10 +297,7 @@ export default (state = initialState, action = {}) => {
       return {
         ...state,
         isFetching: false,
-        errors: [],
-        // After a successful checkout, we may ask again to enable zero waste
-        // https://github.com/coopcycle/coopcycle-app/issues/1824
-        shouldAskToEnableReusablePackaging: true,
+        errors: []
```

Comment:
I don't understand how it can work without that code. It should reset that flag somewhere; otherwise, there would be no prompt next time.

If I'm to do it, I'd put inside `case INIT_SUCCESS:` instead (`CHECKOUT_SUCCESS` is actually a misleading name currently, as it actually means `ANY_CART_REQUEST_SUCCESS`)

Reply:
hi

thanks for testing, i think i tested to quickly. indeed INIT_SUCCESS seems to do the trick